packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -464,6 +464,7 @@ class DropdownButton<T> extends StatefulWidget {
   /// Creates a dropdown button.
   ///
   /// The [items] must have distinct values and if [value] isn't null it must be among them.
+  /// If [items] is null the button is disabled and shows the optional [disabledHint] and the down arrow is drawn as disabled
```

Comment:
nit: please wrap comments to about 80 characters and finish sentences with a period.

Reply:
Consider something like:
```
  /// If [items] is null, the button will be disabled, the down arrow will be grayed out, and
  /// the [disabledHint] will be shown (if provided).
```

Reply:
done